app/src/main/java/com/guruthedev/instagram/applicationClass/ApplicationInitialize.kt

```diff
@@ -0,0 +1,25 @@
+package com.guruthedev.instagram.applicationClass
+
+import android.app.Application
+import com.guruthedev.instagram.preference.Prefs
+
+
+val prefs: Prefs by lazy {
+    ApplicationInitialize.prefs!!
+}
+
+class ApplicationInitialize : Application() {
+    companion object {
+        var prefs: Prefs? = null
+        lateinit var initializer: ApplicationInitialize
```

Comment:
what does this code do, 
```
lateinit var initializer: ApplicationInitialize
 private set
//set means it will perform default initialization
```

Reply:
I removed this Prefs class from that project